tests/test_fdp.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Testing the functions used to control the FDP/FDR
+"""
+import numpy as np
+import pyrft as pr
+from scipy.stats import norm
+
+def test_fdr_bh():
+    """ Testing the fdr_bh function """
+    np.random.seed(10)
+    nvals = 100
+    normal_rvs = np.random.randn(1,100)[0]

```

Comment:
pep8

Reply:
similarly for this comment regarding pep8, though the second 100 should be nvals (which I have changed) perhaps that is what you mean.

Reply:
np.random.randn(1, 100)[0]

Reply:
Likewise this error does not show up on my linter